types/global.d.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-unassigned-import
+import "jest-extended"; // we love the side effects
```

Comment:
This fixes a Typescript check issue where some of the Jest matchers we use (such as `toBeTrue()`) were no longer recognized by Typescript. 

https://khanacademy.atlassian.net/browse/LEMS-2176